src/Applications/GEOSdas_App/fvsetup

```diff
@@ -3874,6 +3862,37 @@ EOF
   }
 }
 
+#======================================================================
+sub get_nens { 
```

Comment:
I think the function name should be more specific, something like "get_nens4ldas()". Otherwise it's very unclear that the only purpose of the function is to count the number of ens member for the purpose of ldas_setup

Reply:
sure we can rename it.